crypto/decrepit/cfb/cfb.c

```diff
@@ -22,6 +22,10 @@
 #include "../../internal.h"
 #include "../../fipsmodule/cipher/internal.h"
 
+// MAXBITCHUNK is used in |aes_cfb1_cipher| to avoid overflow because
+// |AES_cfb1_encrypt| operates data on bit level.
+#define MAXBITCHUNK ((size_t)1<<(sizeof(size_t)*8-4))
```

Comment:
How does this avoid overflow?

Reply:
My understanding of original code is (-4) here ensures there is the space for computing len * 8